CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1281,7 +1281,7 @@ Sie enthält derzeit %2$lld pins.</target>
       </trans-unit>
       <trans-unit id="Please fix your billing issue as soon as possible using the App Store app in order to keep using Callsheet." xml:space="preserve">
         <source>Please fix your billing issue as soon as possible using the App Store app in order to keep using Callsheet.</source>
-        <target state="translated">Bitte löse dein Zahlungsproblem so schnell du kannst im App Store, um Callsheet weiter nutzen zu können.</target>
+        <target state="translated">Bitte löse dein Zahlungsproblem zeitnah im App Store, um Callsheet weiter nutzen zu können.</target>
```

Comment:
"so schnell du kannst" means "as fast as possible!!!", which I don't think you were going for. "zeitnah" is more chill, like "in the near future".